src/lambda/utils.ts

```diff
@@ -61,10 +66,10 @@ export function getLambdaFileNameFromHandler(configuration: Lambda.FunctionConfi
             throw new Error(`Toolkit does not currently support imports for runtime: ${configuration.Runtime}`)
     }
 
-    const fileName = _(configuration.Handler!)
-        .split('.')
-        .initial()
-        .join('.')
+    const handlerArr = _(configuration.Handler!).split('.')
```

Comment:
do we need lodash here?

Reply:
It was already there when I started and it's needed for the `initial()` and `last()` later. Could go either way; it's handy but definitely not necessary.

Reply:
i've been removing lodash where i can, not even sure if we need it at all. it tends to be harder to debug.